src/FittedImage.js

```diff
@@ -103,12 +107,20 @@ const FittedImage = React.createClass({
     this.setState({
       status: states.LOADED
     });
+
+    if (background || !modern) {
```

Comment:
I see your point, it works because of the [spread operator](https://github.com/alexjoffroy/react-fitted-image/blob/master/src/FittedImage.js#L79) on the `img` tag, which automatically apply the `onLoad` and `onError`.

But on the other hand, I think it will be more consistent to handle load events in only one place. So I prefer your second alternative: I think you only need to remove `onLoad` and `onError` from the `props` object, something like:

```js
const { background, src, onLoad, onError, ...props } = this.props;
```

Is it OK for you ? Thanks for your work btw :)